Sources/MySQL/Error.swift

```diff
@@ -11,7 +11,7 @@ public enum Error: Swift.Error {
     case inputBind(String)
     case outputBind(String)
     case fetchFields(String)
-    case prepare(String)
+    case prepare(String, UInt32)
```

Comment:
Also, if we were to add a `UInt32` being passed here it should be in all of the cases. 

I think a better option would be to add a "last error code" variable to the MySQL Error enum as static or to the database class. 

I think you can use `mysql_error()` to get the status code.